Changelog.md

```diff
@@ -21,7 +21,7 @@ Compiler Features:
  * Yul Optimizer: Simplify the starting offset of zero-length operations to zero.
 
 
-Bugfixes:
+Bugfixes:`````
```

Comment:
Is there really no comment yet complaining about this or was I just blind :-)?

Reply:
Oh boy, looks like I was both fat fingered, and blind.

Reply:
Happens :-).

Reply:
> Is there really no comment yet complaining about this or was I just blind :-)?

Just a race condition :P This got in there as a part of the fix, after my review :)